src/js/state/reducers.js

```diff
@@ -51,8 +51,8 @@ export default {
     }
     return _.set(['pages', action.page, 'editMode'], action.edit, state);
   },
-  [SET_PRIVACY_AGREEMENT]: (state, action) => {
-    return _.set('data.privacyAgreementAccepted', action.privacyAgreementAccepted, state);
+  [SET_PRE_SUBMIT]: (state, action) => {
+    return _.set('data.preSubmitAccepted', action.preSubmitAccepted, state);
```

Comment:
`data.preSubmitAccepted` may need to change if we change `fieldName`? I'm curious how we're going to set this to a variable, because the user could name this field anything they want in the form config, right?